runner/startup.sh

```diff
@@ -20,7 +20,7 @@ function wait_for_process () {
 sudo /bin/bash <<SCRIPT
 mkdir -p /etc/docker
 
-echo "{}" > /etc/docker/daemon.json
+[ ! -f /etc/docker/daemon.json ] && echo "{}" > /etc/docker/daemon.json
```

Comment:
```suggestion
if [ ! -f /etc/docker/daemon.json ]; then
  echo "{}" > /etc/docker/daemon.json
fi
```

We've used the long form in the rest of the file so we should be consistent